src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStore.java

```diff
@@ -103,11 +128,12 @@ public BlobStoreContext getContext() throws IOException {
         try {
             Properties props = new Properties();
 
-            if(StringUtils.isNotBlank(REGION)) {
-                props.setProperty(LocationConstants.PROPERTY_REGIONS, REGION);
+            if(StringUtils.isNotBlank(getRegion())) {
+                props.setProperty(LocationConstants.PROPERTY_REGIONS, getRegion());
             }
 
             return ContextBuilder.newBuilder("aws-s3").credentialsSupplier(getCredentialsSupplier())
+                    .overrides(props)
```

Comment:
This is apparently a fix to #29 which somehow dropped this line (perhaps during merge conflict resolution?) so would be better filed separately and merged early.

Reply:
yep it is a missing code on merged code